.github/workflows/ci.yml

```diff
@@ -87,12 +89,19 @@ jobs:
       - name: Make
         env:
           RUN_LONG_TESTS: ${{ matrix.run_long_tests }}
-          AZURE_STORAGE_ENDPOINT: "127.0.0.1:10000"
+          AZURE_STORAGE_ENDPOINT: "http://127.0.0.1:10000/devstoreaccount1"
           AZURE_STORAGE_ACCOUNT: "devstoreaccount1"
           AZURE_STORAGE_ACCESS_KEY: "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
         run: |
           make
 
+      - name: Upload Azurite logs
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: azurite-logs
+          path: /tmp/azurite/debug.log
```

Comment:
This should be `${{ runner.temp }}/azurite/debug.log` given the directory setting above.